control_plane/storcon_cli/src/main.rs

```diff
@@ -560,14 +560,26 @@ async fn main() -> anyhow::Result<()> {
                 .await?;
         }
         Command::TenantDescribe { tenant_id } => {
-            let describe_response = storcon_client
+            let TenantDescribeResponse {
+                tenant_id,
+                shards,
+                stripe_size,
+                policy,
+                config,
+            } = storcon_client
                 .dispatch::<(), TenantDescribeResponse>(
                     Method::GET,
                     format!("control/v1/tenant/{tenant_id}"),
                     None,
                 )
                 .await?;
-            let shards = describe_response.shards;
+            println!("Tenant {tenant_id}");
+            let mut table = comfy_table::Table::new();
+            table.add_row(["Policy", &format!("{:?}", policy)]);
+            table.add_row(["Stripe size", &format!("{:?}", stripe_size)]);
+            table.add_row(["Config", &serde_json::to_string_pretty(&config).unwrap()]);
```

Comment:
Since we know this is a big JSON blob, maybe we should avoid formatting it as a table column?  It could be its own part of the output after everything else.

Reply:
The reason I'm using the table is so that we get proper indenting for tenant-wide vs per-shard infromation

```
Tenant abc123...
|
|
|
Shards:
|
|
|
```

Wondering we can just use `skip_serializing_if(Option::is_none)` on the underlying struct. That would make the output a lot smaller, actually `{}` for most tenants.